extensions/ql-vscode/src/model-editor/model-alerts/model-alerts-view.ts

```diff
@@ -0,0 +1,67 @@
+import { ViewColumn } from "vscode";
+import type { WebviewPanelConfig } from "../../common/vscode/abstract-webview";
+import { AbstractWebview } from "../../common/vscode/abstract-webview";
+import { assertNever } from "../../common/helpers-pure";
+import { telemetryListener } from "../../common/vscode/telemetry";
+import type {
+  FromDataFlowPathsMessage,
+  ToDataFlowPathsMessage,
+} from "../../common/interface-types";
+import type { App } from "../../common/app";
+import { redactableError } from "../../common/errors";
+import { extLogger } from "../../common/logging/vscode";
+import { showAndLogExceptionWithTelemetry } from "../../common/logging";
+
+export class ModelAlertsView extends AbstractWebview<
+  ToDataFlowPathsMessage,
+  FromDataFlowPathsMessage
+> {
+  public static readonly viewType = "codeQL.modelAlerts";
+
+  public constructor(app: App) {
+    super(app);
+  }
+
+  public async showView() {
+    const panel = await this.getPanel();
+    panel.reveal(undefined, true);
+
+    await this.waitForPanelLoaded();
+  }
+
+  protected async getPanelConfig(): Promise<WebviewPanelConfig> {
+    return {
+      viewId: ModelAlertsView.viewType,
+      title: "Model Alerts",
+      viewColumn: ViewColumn.Active,
+      preserveFocus: true,
+      view: "model-alerts",
+    };
+  }
+
+  protected onPanelDispose(): void {
+    // Nothing to dispose
+  }
+
+  protected async onMessage(msg: FromDataFlowPathsMessage): Promise<void> {
```

Comment:
Can we make a new type that's also just `CommonFromViewMessages` for now instead of reusing `FromDataFlowPathsMessage`